release: Release opentelemetry-metrics-sdk 0.7.2 (was 0.7.1) (#1861)
open-telemetry/opentelemetry-ruby: metrics_sdk/CHANGELOG.md

# Release History: opentelemetry-metrics-sdk

> ### v0.7.2 / 2025-07-01

Reviewer: ```suggestion
### v0.7.2 / 2025-07-03
```

> * FIXED: Enfore the aggregation_temporality as sym for exponential_histogram

Reviewer: ```suggestion
* FIXED: Coerce aggregation temporality to be a symbol for exponential histograms
```
